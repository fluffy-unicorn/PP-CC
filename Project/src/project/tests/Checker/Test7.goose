# Correct fork and join
thread myThread() {
    print 10;
}

function void main() {
    thread a = fork myThread();
    thread b = fork myThread();
    join a;
    join b;
}
