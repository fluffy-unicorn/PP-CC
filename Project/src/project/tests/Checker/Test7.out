Sprockell 1 says 10
Sprockell 2 says 10
